lib/main/routes/route_utils.dart

```diff
@@ -0,0 +1,82 @@
+
+import 'package:core/core.dart';
+import 'package:core/data/model/query/query_parameter.dart';
+import 'package:core/data/network/config/service_path.dart';
+import 'package:core/utils/app_logger.dart';
+import 'package:jmap_dart_client/jmap/core/id.dart';
+import 'package:jmap_dart_client/jmap/mail/email/email.dart';
+import 'package:jmap_dart_client/jmap/mail/mailbox/mailbox.dart';
+import 'package:tmail_ui_user/features/login/data/extensions/service_path_extension.dart';
+import 'package:tmail_ui_user/features/thread/domain/model/search_query.dart';
+import 'package:tmail_ui_user/main/routes/navigation_router.dart';
+import 'package:universal_html/html.dart' as html;
+
+abstract class RouteUtils {
```

Comment:
why RouteUtils is abstract ?

Reply:
We don't want to create a constructor for this class.